daemon/analytics.c

```diff
@@ -209,6 +217,38 @@ void analytics_report_oom_score(long long int score){
     analytics_set_data(&analytics_data.netdata_config_oom_score, b);
 }
 
+void analytics_log_conf_disabled_alarm(char *name) {
+    char stored_name[100 + 1];
+    snprintfz(stored_name, 100, "|%s|", name);
+
+    if (!strstr(buffer_tostring(netdata_health_conf_alarms_disabled_b), stored_name)) {
+        snprintfz(stored_name, 100, "%s|", name);
```

Comment:
Is really necessary to copy again? Why do not you give `&stored_name[1]` as argument for next function? :thinking: